jihyeon/app.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const cors = require('cors');
+const morgan = require ("morgan");
+const http = require ("http");
+require("dotenv").config();
+
+const { DataSource } = require('typeorm');
+
+const dataSource = new DataSource({
+    type: process.env.DB_CONNECTION,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    username: process.env.DB_USER,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DATABASE
+})
+
+dataSource.initialize()
+    .then(() => {
+       console.log("Data Source has been initialized!")
+   })
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('dev'));
+
+app.get('/ping', function (req, res, next){
+    res.json({message: 'pong'})
+});
```

Comment:
```suggestion
app.get('/ping', function (req, res, next){
    return res.status(200).json({message: 'pong'})
});
```
http request method에 적합한 status code 또한 백엔드에서 API 별로 지정해줄 수 있습니다.